### ISSUE TITLE
Runtime Reconfiguration pattern

### ISSUE THREAD

**Issue by iluwatar** [iluwatar/java-design-patterns#452]
https://msdn.microsoft.com/en-us/library/dn589785.aspx

**Comment by sta-szek** [comment]
Is that pattern or just a reconfiguration technique?
I mean reconfiguration can be acheived by observer pattern, so here observer is a pattern and runtime reconfiguration is just a nice feature.

**Comment by iluwatar** [comment]
It might be implemented with Observer or some other mechanism like polling.

**Comment by npathai** [comment]
A pattern for similar purpose: #230

**Comment by moguzozcan** [comment]
Hi @iluwatar 
The link you provided directs to https://docs.microsoft.com/tr-tr/azure/architecture/patterns/ that now. Could you renew the link?

**Comment by iluwatar** [comment]
The original link seems to have disappeared. Here is one pointer: http://www.lakshmikanth.com/design-patterns-runtime-reconfiguration-pattern/

**Comment by Deathnerd** [comment]
@iluwatar I get an error page when visiting your most recent link

**Comment by iluwatar** [comment]
Yes, same for me. It seems to be broken at the moment. I couldn't find alternative links.

**Comment by iluwatar** [comment]
Here's link to PDF that describes the pattern: https://download.microsoft.com/download/B/B/6/BB69622C-AB5D-4D5F-9A12-B81B952C1169/CloudDesignPatternsBook-PDF.pdf
